api/rs/slint/Cargo.toml

```diff
@@ -74,6 +74,13 @@ renderer-software = ["i-slint-core/swrenderer", "i-slint-backend-selector/render
 ## This feature is an alias for `backend-qt`
 renderer-qt = ["backend-qt"]
 
+## This feature enables debug output to be sent via the [deferred formatting framework](https://defmt.ferrous-systems.com).
+## Use this in MCU environments where defmt is supported.
+defmt = ["i-slint-core/defmt"]
```

Comment:
Why do we need this within `slint`.  Can't the crates link against defmt themselves?

Reply:
Sure they can, but this is about Slint's debug output. Our internal `debug_log!` that's also used by `debug()` in .slint files.